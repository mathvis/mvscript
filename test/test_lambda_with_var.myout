Expr (LambdaApplication (LambdaFunc [(VarIdentifier "a",IntT)] (Just (Block FunctionBlock []))) (VarIdentifier "a"))
