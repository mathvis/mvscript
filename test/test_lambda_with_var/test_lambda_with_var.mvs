let b = 0;
((a: int): {})(b)
