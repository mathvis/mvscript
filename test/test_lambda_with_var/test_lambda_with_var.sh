#!/bin/bash
helix test_lambda_with_var.mvs test_lambda_with_var.out --vsplit
