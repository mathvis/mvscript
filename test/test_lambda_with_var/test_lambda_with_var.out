Decl (Variable (VarIdentifier "b") (Just IntT) (Just (Type (Int 0))))
Expr (LambdaApplication (LambdaFunc [(VarIdentifier "a",IntT)] (Just (Block FunctionBlock []))) (VarIdentifier "b"))
